core/src/main/java/com/cryptape/cita/protocol/system/CITAjSystemContract.java

```diff
@@ -604,45 +678,33 @@ public QueryResourceResult queryResource(String fromAddr, String permissionAddr)
     /**
      * new group
      * @see <a href="https://docs.citahub.com/zh-CN/cita/sys-contract-interface/interface#newgroup">newGroup</a>
-     * @param superAdminAddress the address of super_admin
      * @param groupName the name of group to be created
      * @param accounts accounts added to the group
-     * @param adminPrivateKey the private key of super_admin
+     * @param adminPrivatekey the private key of super_admin
      * @param version
      * @param chainId
      * @return the transaction hash for creating group
      * @throws IOException
      */
-    public String newGroup(String superAdminAddress,
-                           String groupName,
-                           List<String> accounts,
-                           String adminPrivateKey,
-                           int version,
-                           BigInteger chainId) throws IOException {
-        // account addresses convert to Address object list
-        List<Address> addresses = new ArrayList<>(accounts.size());
-        for(String acc : accounts){
-            addresses.add(new Address(acc));
-        }
-
-        // groupName string convert to bytes32
+    public String newGroup(String groupName, List<String> accounts, String adminPrivatekey, int version, BigInteger chainId) throws Exception {
```

Comment:
`adminPrivatekey` should be `adminPrivateKey`, keep the same naming style as previous